iml-agent/systemd-units/rust-iml-agent.service

```diff
@@ -13,6 +13,8 @@ Environment=CRT_PATH=/etc/iml/self.crt
 Environment=PFX_PATH=/etc/iml/identity.pfx
 Environment=AUTHORITY_CRT_PATH=/etc/iml/authority.crt
 Environment=LTUER_CONF_PATH=/etc/iml/ltuer.conf
+# Here {name} corresponds to the field names of iml_agent::action_plugins::lamigo::Config
+Environment=LAMIGO_UNIT_PATH=/etc/systemd/system/lamigo-{fs}-{mdt}.service
```

Comment:
It will be easy to break by accident. And since this path is for systemd and not for IML, I don't think it should be here. @mpankov

Reply:
Yep, let's discuss it, if it feels not right.